src/module.js

```diff
@@ -835,10 +830,6 @@ export default class DailyIframe extends EventEmitter {
 
   setSubscribeToTracksAutomatically(enabled) {
```

Comment:
does it not makes sense to add the `avatar` option here? actually, probably should've asked earlier haha...what is the use case that is motivation this avatar-bar-only subscription option?